README.md

```diff
@@ -9,9 +12,8 @@ An SQLite3 library with an Ecto adapter implementation.
 * Prepared statements are not immutable. You must be careful when manipulating
   statements and binding values to statements. Do not try to manipulate the
   statements concurrently. Keep it isolated to one process.
-* Adding a `CHECK` constraint is not supported by the Ecto adapter. This is due
-  to how Ecto handles specifying constraints. In SQLite you must specify the
-  `CHECK` on creation.
+* Asynchronous writing is not supported by SQLite3 and will not be supported
```

Comment:
Asynchronous testing?

Reply:
Maybe a better word would be "Simultaneous" ?